src/video_core/renderer_vulkan/vk_rasterizer.cpp

```diff
@@ -51,13 +51,24 @@ void Rasterizer::Draw(bool is_indexed, u32 index_offset) {
     BeginRendering();
     UpdateDynamicState(*pipeline);
 
+    u32 instance_offset = 0;
```

Comment:
Can you add this and vertex offset as a method function to Shader::Info? Will probably allow this to be a bit cleaner